README.md

```diff
@@ -1,9 +1,47 @@
-## To Connect to Databases Locally
+# NC NEWS
 
-In order to successfully connect to the two databses locally you need to create these two files:
-.env.test
+## Hosted API
+
+### To use visit: https://ncnews.cyclic.app/
+
+
+## Summary 
+
+This project is an API to get news articles, post comments to a new article etc. To view what requests you can make, visit https://ncnews.cyclic.app/api which will show you all available endpoints and a description of their response.
+
+## **WARNING**
+
+ To run this project you will need Node.js v18.10.0 and Postgres v8.7.3.
+
+## Instructions 
+
+## 1. Clone Repo
+
+Fork this repository: https://github.com/farwah1/nc-news-public and clone down to your local machine.
+
+## 2. Install Dependencies
```

Comment:
Just running npm i will install all dependencies listed in your package files, no need to install one-by-one!